src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -122,6 +122,7 @@ const TopicFragment = ({
     const renderHeader = (data, index) => {
       return (
         <DiscoveryTitleSeparator
+          withBorderBottom={true}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `withBorderBottom` prop is a good way to conditionally render the bottom border. However, it's important to ensure that this prop is also handled properly in the `DiscoveryTitleSeparator` component. If not, it could lead to unexpected UI behavior.
```diff
+ withBorderBottom={true}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->